Testing/ORKTest/ORKTest/TaskFactory.h

```diff
@@ -0,0 +1,93 @@
+//
+//  TaskFactory.h
+//  ORKTest
+//
+//  Created by Ricardo Sanchez-Saez on 1/9/17.
+//  Copyright © 2017 ResearchKit. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#define DefineStringKey(x) static NSString *const x = @#x
+
+DefineStringKey(ConsentTaskIdentifier);
```

Comment:
That's a great idea. Do you think it's better to do it in a follow-up PR (to make this one easier to review)?